### PR TITLE
Issue509 fix readmes to use ExampleConfig

### LINKED ISSUE #509: Update all READMEs to use ExampleConfig
This issue is in the same spirit as https://github.com/oceanprotocol/ocean.py/pull/508 but applied to all of the example flow READMEs.

By doing this, it reduces the risk that users will connect to ganache with `block_confirmations > 0` which causes infinite loops because ganache doesn't auto-mine blocks. For additional context, see https://github.com/oceanprotocol/ocean.py/issues/502

### PR DESCRIPTION
**Pull request by DMats**
Closes #509 .

Changes proposed in this PR:

- Fix all example flow READMEs to use `ExampleConfig`
- Fix all `metadata_cache_uri` examples that referenced the old network-speicific Aquarius URLs.
- Add `block_confirmations` to `config.ini`
- Remove `Config.SECTION_UITL` constant. Vestige of past design.

### CODE REVIEW

File: READMEs/parameters.md [hunk 1]
```diff
@@ -7,58 +7,45 @@ SPDX-License-Identifier: Apache-2.0
...
 Here are examples.
 
-## 1. dict input, filled from envvars
+## 1a. Config object input, filled from ExampleConfig (recommended)
 
 First, in console:
 
 ```console
 export OCEAN_NETWORK_URL=https://rinkeby.infura.io/v3/<your Infura project id>
-export METADATA_CACHE_URI=https://aquarius.rinkeby.oceanprotocol.com
```
Comment (mariacarmina) [inline READMEs/parameters.md:24]: Here you deleted the exports of `METADATA_CACHE_URI` and `PROVIDER_URL` and for `Unsetting envvars` section you basically unset something that it's not mentioned at the beginning, maybe we can keep those methods as we had before and just add another method for config creation such as ExampleConfig to include all of the aspects.

**Review by mariacarmina — COMMENTED** [review]
Overall, it looks fine and it's good that we provide consistency to all the READMEs. I have mixed feelings about the changes from the `parameters.md` README and I'll let others to share their thoughts as well @trentmc @calina-c .

**Comment by trentmc** [comment]
> Overall, it looks fine and it's good that we provide consistency to all the READMEs. 
Yes, ok.

> I have mixed feelings about the changes from the parameters.md
Agreed. The point of parameters.md is to teach the users how parameters work, not to hide it in some included file. Recommend  to explain both options: (a) how the full config.ini way works, with the worked example of what was there beforem and (b) how the import-way works.

**Review by trentmc — CHANGES_REQUESTED** [review]
> Overall, it looks fine and it's good that we provide consistency to all the READMEs. 

Yes, ok.

> I have mixed feelings about the changes from the parameters.md

Agreed. The point of parameters.md is to teach the users how parameters work, not to hide it in some included file. Recommend  to explain both options: (a) how the full config.ini way works, with the worked example of what was there beforem and (b) how the import-way works.

**Comment by DMats** [comment]
Thank you @mariacarmina and @trentmc for reviewing!

Based on your feedback, I believe that the `parameters.md` file needs more changes to fully explain all the configuration methods. A description of the needed changes and the changes made thus far can be found in the new Draft PR: https://github.com/oceanprotocol/ocean.py/pull/511.

I don't want to block the updates to the other READMEs so I've cherry-picked the `parameters.md` changes to a new branch and reverted them in *this* branch. 

Please approve at your nearest convenience.

File: READMEs/parameters.md [hunk 2]
```diff
@@ -7,45 +7,58 @@ SPDX-License-Identifier: Apache-2.0
...
+An `Ocean` instance will hold a `Config` instance that holds various config parameters. These parameters need to get set. This is set based on what's input to `Ocean` constructor:
 
-1. Config object input
-    * Filled from ExampleConfig: `Ocean(ExampleConfig.get_config())` (recommended)
-    * Filled from config file: `Ocean(Config('config.ini'))`
-    * Filled from config file specified by `OCEAN_CONFIG_FILE` envvar: `Ocean(Config())`
-    * Filled from config dict: `Ocean(Config('network':..}))`
-2.  dict input: `Ocean({'network':..})` (deprecated because it doesn't support all config options)
+1.  dict input: `Ocean({'network':..})`
+2.  Config object input: `Ocean(Config('config.ini'))`
```
Comment (mariacarmina) [inline READMEs/parameters.md]: For the second point `Config object input` we could let the unordered list which described each method of config creation to be clearer for the user. That's just my opinion.

**Review by mariacarmina — APPROVED** [review]

File: READMEs/parameters.md (hunk 2 above)
Comment (DMats) [reply READMEs/parameters.md]: Thanks for the comment!  

All of the changes to parameters.md were reverted and moved over to #511 so I'll take the liberty of copying this comment over there for you.

Comment (DMats) [reply READMEs/parameters.md]: On second thought, the `Config object input` line doesn't exist in #511 so I'm unable to copy this for you.

**Review by trentmc — APPROVED** [review]

**Pull request merged**